pypika/tests/test_inserts.py

```diff
@@ -111,6 +111,50 @@ def test_insert_null(self):
         self.assertEqual('INSERT INTO "abc" VALUES (NULL)', str(query))
 
 
+class PostgresInsertIntoOnConflictTests(unittest.TestCase):
+    table_abc = Table('abc')
+
+    def test_insert_on_conflict_do_nothing_field(self):
+        query = PostgreSQLQuery.into(self.table_abc).insert(1).on_conflict_do_nothing(self.table_abc.id)
```

Comment:
Well it makes more sense. BTW, on do_update, as far as I know it can not be done like your comment. Besides the '==' makes it as a condition, the variables to the method should be as kwargs, not objects. The way I changed it now, it is like the default intsert/update method, one field per call

Reply:
okay, that works too. Thanks for updating.